src/main/kotlin/lotto/generator/ticket/ActualLottoShop.kt

```diff
@@ -0,0 +1,8 @@
+package lotto.generator.ticket
+
+import lotto.domain.LottoTickets
+
+object ActualLottoShop : LottoShop {
+    override fun provideAutoTickets(ticketCount: Int): LottoTickets = AutoTicketGenerator.create(ticketCount)
+    override fun provideManualTickets(ticketCount: Int): LottoTickets = ManualTicketGenerator.create(ticketCount)
```

Comment:
AutoTicketGenerator, ManualTicketGenerator를 싱글턴으로 접근하는구조네요! 
싱글턴 패턴은 자주 사용되는 패턴이지만, 잘못사용되면 객체간 결합도를 높여주게 되는 현상이 있어요
```suggestion
class ActualLottoShop(
    autoTicketGenerator: TicketGenerator,
    manualTicketGenerator: lTicketGenerator
) : LottoShop {
    override fun provideAutoTickets(ticketCount: Int): LottoTickets = autoTicketGenerator.create(ticketCount)
    override fun provideManualTickets(ticketCount: Int): LottoTickets = manualTicketGenerator.create(ticketCount)
```
위 처럼 바꾼 코드랑은 어떤점이 다를까요? 고민해보시면 좋을거같아요!